### ISSUE TITLE
RCAF CT-155 Hawk x17

### ISSUE THREAD

**Issue by cdnice55** [PygmalionOfCyprus/cmo-db-requests#193]
### DB Selector

DB3K

### Hypothetical

No

### Name

CT-155 Hawk

### Country

Canada

### Service

Royal Canadian Air Force

### In Commission

2000-xxxx

### Length (m)

12.42

### Wingspan (m)

9.08

### Height (m)

3.98

### Empty Weight (kg)

4400

### Maximum Takeoff Weight (MTOW) (kg)

9,100

### Payload Weight (kg)

3084

### Crew

2

### Propulsion

Rolls-Royce Mk.871 Adour turbofan

### Rate of Climb (ft/min)

9240/27710

### Service Ceiling (ft)

45000

### Military/Afterburner Speed (kt)

510

### Cruise Speed (kt)

480

### Supercruise

No

### Ferry Range (nm)

2000

### Takeoff / Landing Distance (m)

451-900m TOD/LAD

### Fuel Capacity (kg)

1305

### Sensors

1x Mk1 Eyeball

### Mounts

130 ImpG Drop Tank
190 ImpG Drop Tank

### Loadout Options

- Ferry DB ID #15099
- Maintenance
- Reserve
- Basic Flight Training (Training) DB ID 15100

### Comms/Datalinks

UHF/VHF Radio [Unsecure]

### Aircraft Codes

Night Navigation (Ferry, Air-to-Air, Air-to-Surface Missiles)

### Comments

Basically a copy of Hawk T.2 (UK) minus the Exocet training loadout

### Sources

http://www.rcaf-arc.forces.gc.ca/en/aircraft-current/ct-155.page
https://en.wikipedia.org/wiki/BAE_Systems_Hawk#Former_operators
https://army.ca/wiki/index.php/CT-155_Hawk

![ct-155-hawk-fact-sheet-image](https://user-images.githubusercontent.com/99632305/154366247-eea84a00-9501-4bae-930a-e63faaf07385.png)
